sdk/tests/wallet/syncing.rs

```diff
@@ -70,7 +70,7 @@ async fn sync_only_most_basic_outputs() -> Result<()> {
                 UnlockCondition::Expiration(ExpirationUnlockCondition::new(
                     account_1_address,
                     // Already expired
-                    account_0.client().get_time_checked().await? - 5000,
+                    account_0.client().get_slot_index().await? - 5000,
```

Comment:
This can easily underflow now

Reply:
Well yeah but not a big deal in a test, we'll put proper values when we know more, or what do you advise?

Reply:
Yeah fine to update later